CMakeLists.txt

```diff
@@ -4,9 +4,8 @@
 cmake_minimum_required(VERSION 2.8)
 project(zeq)
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMake
-  ${PROJECT_SOURCE_DIR}/CMake/common)
+  ${CMAKE_SOURCE_DIR}/CMake/common)
```

Comment:
Could you exaplain me why it has to be the top level CMake/common (I guess the superproject one)?
Does this mean all subprojects must be prepared to use the same CMake/common commit to work?